sampletester/testplan.py

```diff
@@ -263,34 +267,30 @@ def accept(self, visitor: Visitor):
     return visitor.end_visit()
 
 
-def suite_configs_from(test_files):
-  """Returns the suite configs (key/value pairs) from all the `test_files`.
+SCHEMA_TYPE_VALUE = 'test'
+SCHEMA_SUBTYPE_VALUE = 'samples'
```

Comment:
This variable doesn't seem to be used.

Reply:
Good catch. Using it now.

Reply:
I kinda feel `SCHEMA_TYPE_VALUE = "test/samples"` is good enough, providing the variable `SCHEMA_TYPE_VALUE` is the expected value of the `type` field in test config or manifest yamls.

Like for example, if I were to depend on this module, I would need to look at the code to know that actual value of the `type` field is the combination of two parts.